paper-server/src/main/java/org/bukkit/craftbukkit/entity/CraftHumanEntity.java

```diff
@@ -801,6 +806,53 @@ public boolean dropItem(boolean dropAll) {
         // Paper end - Fix HumanEntity#drop not updating the client inv
     }
 
+    @Override
+    public @Nullable Item dropItem(final @NotNull ItemStack itemStack, final boolean throwRandomly) {
+        final int slot = this.inventory.first(itemStack);
+        if (slot == -1) {
+            return null;
+        }
+
+        return this.dropItem(slot, throwRandomly);
+    }
+
+    @Override
+    public @Nullable Item dropItem(final int slot, final boolean throwRandomly) {
```

Comment:
Do we have to do some math to convert this "slot" to a bukkit slot number? I remember them being different in the inventory API

Reply:
From my testing it seemed to work fine, though if you provide me with more information, I could very quickly check it

Reply:
I think its fine actaually, the slot is only interacting with the bukkit inventory system `this.inventory` is a craftbukkit inventory. So it should be fine.